src/app/Components/home/home.component.ts

```diff
@@ -75,6 +75,7 @@ export class HomeComponent {
         this.speechRecognitionService.recognize((result) => {
             this.commandTextValue = result;
             this.srState = 'inactive';
+            this.sendCommand(this.commandTextValue);
```

Comment:
use `result` here because it's more intuitive

Reply:
Yeah you're right, it's better access-wise